pkg/config/analysis/local/context_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package local
+
+import (
+	"testing"
+	"time"
+
+	. "github.com/onsi/gomega"
+
+	"istio.io/istio/pkg/config/schema/collection"
+)
+
+func TestIstiodContextTimeout(t *testing.T) {
```

Comment:
This test is flaky (`111 runs so far, 15 failures (88.10% pass rate). 1.949718985s avg, 3.644403612s max, 677.973719ms min`). using time.Sleep is not a great idea in tests

Reply:
Fixed